pkg/minishift/config/validations.go

```diff
@@ -31,18 +32,19 @@ import (
 
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	minishiftConstants "github.com/minishift/minishift/pkg/minishift/constants"
-	"github.com/minishift/minishift/pkg/minishift/shell/powershell"
 	"github.com/minishift/minishift/pkg/util"
 	stringUtils "github.com/minishift/minishift/pkg/util/strings"
 )
 
 func IsValidDriver(string, driver string) error {
 	if runtime.GOOS == "windows" {
-		posh := powershell.New()
-		defer posh.Close()
-		checkWindowsName := `(Get-WmiObject -Class Win32_OperatingSystem).Caption`
-		stdOut, _ := posh.Execute(checkWindowsName)
-		if driver == "hyperv" && strings.Contains(stdOut, "Windows 7") {
+		cmd := exec.Command("wmic", "os", "get", "Caption", "/value")
+		stdOut, err := cmd.Output()
+		if err != nil {
+			return err
+		}
+		out := fmt.Sprintf("%s", stdOut)
```

Comment:
why not use `stdOut` in the `string.Contains()` ?

Reply:
`stdOut` is of type `[]byte` so need to convert it to be able to use in `strings.Contains()`
```
cannot use stdOut (type []byte) as type string in argument to "strings".Contains
```

Reply:
no `.String()` ?

Reply:
no `.String()` however i haven't tried using `string()`.

Reply:
Weird... if i modify the code as following:
```go
if driver == "hyperv" && strings.Contains(string(stdOut), "Windows 7") || strings.Contains(string(stdOut), "Windows XP") {
			return fmt.Errorf("Driver '%s' is not supported", driver)
		}
```
It throws an error: 
```
pkg/minishift/config/validations.go:46:51: cannot call non-function string (type string)
make: *** [Makefile:86: /home/xxx/work/bin/minishift] Error 2

```

But the same code runs and builds without error if its done like this:
```go
package main

import "fmt"
import "os/exec"
import "strings"


func main() {
	c := exec.Command("echo", "hello", "world")
	out,_ := c.Output()
	fmt.Println(strings.Contains(string(out), "hello"))
}
```
@gbraad What am i doing wrong?